cEP-0032.md

```diff
@@ -0,0 +1,651 @@
+| Authors  | Keshav Garg <mailto:co16326.ccet@gmail.com>           |
+| Status   | Work In Progress                                      |
+| Type     | Feature                                               |
+
+## Abstract
+
+This document describes the features that will increase the usability and
+easiness for all the open-source enthusiasts by enhancing and improving the
+coala Community website as well as coala Projects website for newcomers,
+developers as well as for other communities. In addition, new API Endpoints
+will be created in coala webservices. These new endpoints will manage the user
+-database as well as coala database in a more structured form.
+
+## Introduction
+
+A website plays an essential role for an open-source community because it
+spreads the idea of offerings that community provide to other organizations,
+with information about the active community by exploring their skills as well
+as their precious contributions to the organization. And to provide such
+offerings this project aims to provide useful features to community members as
+well as to other developers and organizations. They can use the below
+mentioned features to be a part of coala community and can even get information
+about coala community and its contributors who have been part of SoC programs.
+
+## coala [Community website](community.coala.io/)
+
+### Enhancements
+
+The listed enhancements will ease the navigatibility and usability for
+newcomers, developers and to all others who wanted to be a part of coala
+organization. It will help community members to know about their performance
+in-comparison to other members. Following are the new enhancments that will be
+added to coala community website:
+
+1. **Display a webpage for-**
+   1. **organization teams**
+
+        This will show information about all [coala teams](https://github.com/orgs/coala/teams).
+        It will definitely not show what permissions are available to team members
+        if a developer is a part of that team. But, it will definitely provide
+        information like *What does this team aim to do?*, *Members count* etc.
+        Clicking on a **team name** will redirect the user to either on GitHub
+        showing all team members or a new webpage which will show all the team
+        members of this coala team.
+
+   2. **contributors' geolocation in form of a map**
+
+        Showing the *global influence of an organization* around the world of open-
+        source organization can be achieved by showing the contributors' geolocation.
+        It can be shown in the form a world map which shows the geolocaton of all
+        contributors that are members of an organization or have been contributing
+        to the organization.
+
+   3. **mentors**
+
+        It will be good if community website can show all the potential mentors that
+        organizaton has. It will allow other organization to get to know more about
+        them.
+
+2. **Add forms for -**
```

Comment:
I am skeptic about this section too.

Not sure if we need all these forms, needed @jayvdb confirmation.

Reply:
See @jazzband